arc/data-challenge/build-dummy-predictions.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Any value in creating a shared script ? Right now this is repeated in three different places.  Alternatively, we could add the last command to the README instead since I don't think anyone will be running this script explicitly. Its more to describe the provenance of the dummy predictions file.

Reply:
Hmm, not much value in sharing this short script in my opinion. Also, they're not exactly the same scripts, because they extract different JSONL files from different zip files, so that would have to be factored out as args or something.

For simplicity I'd like to keep the the data sub-directories as un-entangled as possible.